test/unit/jasmine-boot.js

```diff
@@ -38,33 +38,31 @@
 
 // Modified jasmine's boot.js file to load PDF.js libraries async.
 
-'use strict';
```

Comment:
The actual test files are being parsed by SystemJS, which will add a `'use strict';` directive to each module, but unless I'm mistaken *this* file runs as-is in the browser.
Hence it may be a good idea to keep this, such that this file still runs in strict mode.